triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -204,6 +207,11 @@ object Server {
     }
   }
 
+  case class TriggerActorWithParty(
```

Comment:
Something like `RunningTrigger` might be a better name. I wouldn’t be surprised if you end up adding more metadata here.